cardano-node/src/Cardano/Tracing/OrphanInstances/Consensus.hs

```diff
@@ -674,11 +674,17 @@ instance ( ConvertRawHash blk
           <> "."
         ImmDB.ChunkValidationEvent e -> case e of
           ImmDB.StartedValidatingChunk chunkNo outOf ->
-               "Validating chunk no. " <> showT chunkNo <> " out of " <> showT outOf
-            <> ". Progress: " <> showProgressT (max (chunkNoToInt chunkNo - 1) 0) (chunkNoToInt outOf) <> "%"
+            | chunkNo == outOf ->
+                "Validating last chunk no. " <> showT chunkNo <> " out of " <> showT outOf <> "."
+            | otherwise ->
+                   "Validating chunk no. " <> showT chunkNo <> " out of " <> showT outOf
+                <> ". Progress: " <> showProgressT (chunkNoToInt chunkNo) (chunkNoToInt outOf + 1) <> "%"
           ImmDB.ValidatedChunk chunkNo outOf ->
-               "Validated chunk no. " <> showT chunkNo <> " out of " <> showT outOf
-            <> ". Progress: " <> showProgressT (chunkNoToInt chunkNo) (chunkNoToInt outOf) <> "%"
+            | chunkNo == outOf ->
+                "Validated last chunk no. " <> showT chunkNo <> " out of " <> showT outOf <> "."
```

Comment:
Do we need these cases? It might be nice to show the `100%` progress bar instead of this message

Reply:
Not sure I understand, this should print "Validated chunk 100 out of 100. Progress: 100%"

Reply:
Actually no. You are right. Let me amend it.

Reply:
Done